exercises/exercise5/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+)
+
+type Node struct {
+	Value int
+	Next  *Node
+}
+
+func Prepend(head **Node, x int) {
+	// Plan:
+	// * Create new head
+	// * Make current head new head's next
+	// * Addign new head to current head pointer
+
+	var newHead *Node = new(Node)
```

Comment:
You don't need to specify the type twice: `newHead := new(Node)` would be enough